### PR TITLE
Add some basic installation instructions to the readme

### PR DESCRIPTION
**Pull request by micgro42**
That way, maybe it won't take the next person 3 days to figure out how to use this package 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
...
+
+* Python 3.5, 3.6 or 3.7
+* All the system requirements of [revscoring](https://github.com/wikimedia/revscoring)
+
+### Installation steps
+
+1. clone this repository
+2. install its dependencies `python -m pip -r requirements.txt`
+3. install the package itself `python setup.py install`
+4. Now you can run `make` to build the english wikipedia article quality model or `make wikidatawiki_models` to build the item quality model for wikidata
```
Comment (halfak) [inline README.md:42]: This isn't really an installation step.  #4 should be moved to a section called "building models" or something like that.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
                      'ga': 0.21617801281707663,
                      'start': 0.13906813268582238}}
 
+## Install
+
+### Requirements
+
+* Python 3.5, 3.6 or 3.7
```
Comment (halfak) [inline README.md:34]: We also support python 3.8.

File: README.md (hunk 1 above)
Comment (halfak) [reply README.md:42]: Oh also, when you run "make", it should build all of the models -- not just the English Wikipedia one.

File: README.md [hunk 3]
```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
...
+* All the system requirements of [revscoring](https://github.com/wikimedia/revscoring)
+
+### Installation steps
+
+1. clone this repository
+2. install its dependencies `python -m pip -r requirements.txt`
```
Comment (halfak) [inline README.md:40]: Shouldn't this command just be `pip install -r requirements.txt`

File: README.md [hunk 4]
```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
...
+3. install the package itself `python setup.py install`
```
Comment (halfak) [inline README.md:41]: If you run `python setup.py install` it will automatically try to gather the requirements that the pip command above is referencing.

**Comment by halfak** [comment]
Nice work.  I left a couple of notes.  I imagine the biggest hurdle was the installation and setup of revscoring, is that right?

File: README.md (hunk 1 above)
Comment (micgro42) [reply README.md:42]: Yes, I intended them more as a way to verify that the installation worked. I'll update the wording to reflect that

File: README.md (hunk 2 above)
Comment (micgro42) [reply README.md:34]: I'm not sure that this is robustly true. Support for Python 3.8 was added in [SciPy 1.3.2](https://github.com/scipy/scipy/releases/tag/v1.3.2) (see also scipy/scipy#10927) and revscoring's SciPy requirement is `scipy >= 1.2.3, < 1.2.999`. In practice, that means that some Fortran compilation step in SciPy 1.2 fails with Python 3.8, but succeeds with Python 3.7.

File: README.md (hunk 3 above)
Comment (micgro42) [reply README.md:40]: `python -m pip` is the explicitly recommended way:
* [official docs](https://docs.python.org/3/installing/index.html#basic-usage)
* [more reasoning by a python core dev](https://snarky.ca/why-you-should-use-python-m-pip/)

File: README.md (hunk 4 above)
Comment (micgro42) [reply README.md:41]: Mh, I thought that didn't work when I tried it previously; but I attempted it again just now on a fresh git checkout and it works just fine. Maybe I didn't have the correct python version or a virtual env previously?

Anyway, I will remove the 2. step. Thank you for pointing it out.

**Comment by micgro42** [comment]
> [...] I imagine the biggest hurdle was the installation and setup of revscoring, is that right?

That and while I remembered (It has been quite some years since I last worked with Python) to do something like `pip install -r requirements.txt`, I had forgotten about `python setup.py install`. So I ran into strange "articlequality not found" errors based on which I tried `pip install articlequality` which made the errors go away but installed a hopelessly outdated version of articlequality (In retrospect, it is very strange that it worked). But in the end and with some friendly pointers I got it to run as intended.

Anyway, thank you for your feedback :)

File: README.md (hunk 2 above)
Comment (halfak) [reply README.md:34]: Interesting!  Our test suite all passes with python 3.8.  It's part of our CI pipeline.

File: README.md (hunk 3 above)
Comment (halfak) [reply README.md]: That's so strange, but it's fair.  The author's argument that "you don't run pip all of the time" is very wrong in my experience, but I think the point still stands.

**Comment by halfak** [comment]
This looks great.  thanks for your work.

**Pull request merged**